packages/api/src/schema/api-schema.yaml

```diff
@@ -162,6 +162,20 @@ components:
             - VP8
             - VP9
           example: H.264
+    webhook-payload:
```

Comment:
yeah that is correct, create webhook also uses this payload, but we can't use webhook, since it has many other param which user can't/shouldn't update like userId

Reply:
Ok, as long as we use it on the code as well, not only the schema, this is fine!